### PR TITLE
Implementation of Fernet keys rotation for Keystone

### PR DESCRIPTION
**Pull request by katrybacka**
This PR introduces Fernet keys rotation that are the base for creating and validating tokens by Keystone. This rotation should work like that:

- it should change staged key to primary key (primary key is the one with the highest index)
- primary becomes secondary (since after first step it does not have the highest index)
- new staged key needs to be generated as key with index "0"
- number of keys cannot exceed maxActiveKeys with is computed by controller using time of token expiration, allow token expired window and rotationInterval.

All of the mentioned fields have default values.

### CODE REVIEW

**Comment by katrybacka** [comment]
recheck ha

**Comment by katrybacka** [comment]
recheck ha

File: pkg/controller/keystone/keystone_controller.go [hunk 1]
```diff
@@ -76,6 +75,14 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Watch for changes to secondary resource Postgres and requeue the owner Keystone
```
Comment (mfigurski) [inline pkg/controller/keystone/keystone_controller.go:78]: Postgres? 🙃

Comment (katrybacka) [reply pkg/controller/keystone/keystone_controller.go]: done

**Comment by katrybacka** [comment]
recheck ha

**Comment by katrybacka** [comment]
recheck

**Pull request merged**